cypress/integration/errorPagePersian.js

```diff
@@ -1,5 +1,5 @@
 import config from '../support/config';
-import describeForLocalOnly from '../support/describeForLocalOnly';
+import describeForLocalAndTest from '../support/limitEnvRuns';
```

Comment:
```suggestion
import describeForLocalOnly from '../support/limitEnvRuns';
```
Blocked pending this work: https://github.com/bbc/simorgh-infrastructure/issues/347